target/linux/bmips/dts/bcm6328-dlink-dsl-2750b-b1.dts

```diff
@@ -0,0 +1,276 @@
+#include "bcm6328.dtsi"
+
+/ {
+	model = "D-Link DSL-2750B rev B1/DSL-2740B rev F1/DSL-2741B rev F1";
+	compatible = "dlink,dsl-2750b-b1", "brcm,bcm6328";
+
+	aliases {
+		led-boot = &led_power_green;
+		led-failsafe = &led_power_red;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_green;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <100>;
+
+		wifi {
+			label = "wifi";
+			gpios = <&gpio 10 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WLAN>;
+			debounce-interval = <60>;
+		};
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 23 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+			debounce-interval = <60>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio 24 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led@31 {
+			function = LED_FUNCTION_USB;
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&gpio 31 GPIO_ACTIVE_LOW>;
+			trigger-sources = <&ehci_port1>;
+			linux,default-trigger = "usbport";
+		};
+	};
+
+	ath9k-leds {
+		compatible = "gpio-leds";
+
+		wlan {
```

Comment:
`wlan@8`

Reply:
fixed

Reply:
gpio-leds doesn't have `reg` property and pin number is not an address, so `-` should be used instead of `@`.

https://elixir.bootlin.com/linux/v6.6.47/source/Documentation/devicetree/bindings/leds/leds-gpio.yaml

Reply:
> gpio-leds doesn't have `reg` property and pin number is not an address, so `-` should be used instead of `@`.

You're totally right and I had no idea about it... >.<
@a-gave please, change all leds from `led@` to `led-`.
I will take care of changing all bmips DTS in the near future.

Reply:
Note: BCM LEDs under `&leds { }` have `reg` property and should be left as-is.

Reply:
fixed (changed then wlan@8 to led-8)